snap/snapcraft.yaml

```diff
@@ -210,7 +243,10 @@ apps:
       TMPDIR: $XDG_RUNTIME_DIR
       ALWAYS_USE_PULSEAUDIO: 1
       DISABLE_WAYLAND: 1
-
+      LD_LIBRARY_PATH: $LD_LIBRARY_PATH:$SNAP/graphics/lib
+      LIBGL_DRIVERS_PATH: $LIBGL_DRIVERS_PATH:$SNAP/graphics/dri
+      LIBVA_DRIVERS_PATH: $LIBVA_DRIVERS_PATH:$SNAP/graphics/dri
+      __EGL_VENDOR_LIBRARY_DIRS: $__EGL_VENDOR_LIBRARY_DIRS:$SNAP/graphics/glvnd/egl_vendor.d
```

Comment:
If we're using layouts to bind these to their usual paths under `/usr`, then maybe the environment variables aren't needed?